src/aboutdialog.cpp

```diff
@@ -11,13 +11,18 @@
 
 #include "aboutdialog.h"
 #include "utilsVersion.h"
+#include "utilsSystem.h"
 
 AboutDialog::AboutDialog(QWidget *parent)
 	: QDialog(parent)
 {
 	ui.setupUi(this);
 	ui.textBrowser->setOpenExternalLinks(true);
-    ui.textBrowser->setHtml(QString("<b>%1 %2</b> (git %3)").arg(TEXSTUDIO,TXSVERSION,TEXSTUDIO_GIT_REVISION ? TEXSTUDIO_GIT_REVISION : "n/a") + "<br>" +
+	ui.textBrowser->setOpenLinks(false);
```

Comment:
any reason why not to do it like the txs link, some lines further down ?
```
tr("Project home site:") + " <a href=\"https://texstudio.org/\">https://texstudio.org/</a><br><br>" +
```

Reply:
You mean why I use setOpenLinks(false) and use the signal anchorClicked? Well, I tried the simple approach. But after many tries I ended up with reading the [qt docs](https://doc.qt.io/qt-6/qtextbrowser.html#openExternalLinks-prop). Reason: Links starting with https are considered external links, while those starting with file are internal links. This means that the textBrowser itself opens the file as a html page:

![image](https://user-images.githubusercontent.com/102688820/216762132-852b6557-cde2-46dd-b9e7-8cfd4ccae9a0.png)

You can test this easily by setting lines 21 (setOpenLinks) and 24 (connect) as comments (image shows result).

Even if changelog.txt where correct html, I think no one want's to read the news in such a small window.

Btw: There is a context menu on the image that opens the image in a larger version. I would suggest trying to open the larger version when clicking on the image without using a context menu. What do u think? Maybe in a second PR.